ddtrace/internal/datastreams/processor.py

```diff
@@ -262,10 +262,13 @@ def decode_pathway(self, data):
 
     def decode_pathway_b64(self, data):
         # type: (str) -> DataStreamsCtx
-        binary_pathway = data.encode("utf-8")
-        encoded_pathway = base64.b64decode(binary_pathway)
-        data_streams_context = self.decode_pathway(encoded_pathway)
-        return data_streams_context
+        try:
+            binary_pathway = data.encode("utf-8")
+            encoded_pathway = base64.b64decode(binary_pathway)
+            data_streams_context = self.decode_pathway(encoded_pathway)
+            return data_streams_context
+        except AttributeError:
```

Comment:
If the change above is accepted we can revert this or at least we should change the method signature to `type: Optional[str] -> DataStreamsCtx ` and then handle the case were data is None without raise an Exception.